imagedephi/redact/svs.py

```diff
@@ -82,6 +85,15 @@ def __init__(
                 else:
                     self.no_match_description_keys.add(key)
 
+    def get_associated_image_key_for_ifd(self, ifd: IFD):
+        """Attempt to match an associated image with a rule. Fallback to default rule."""
+        image_description = str(ifd["tags"][270]["data"])
```

Comment:
`270` is magic. Is there any symbolic identifier for this tag?

Reply:
b536a25